docs/clients/multi_turn_angle_control.rst

```diff
@@ -118,69 +118,74 @@ Message Table
+		+--------+---------------------------------+----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 29     | ff                              | get, set       | uint32    |           | Feed forward term                                                                                                                                                                                                                                                                                                                                  |
+		+--------+---------------------------------+----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 30     | sample_zero_angle               | set            |           |           | Sets the module’s current postiion as the zero angle.                                                                                                                                                                                                                                                                                              |
+		+--------+---------------------------------+----------------+-----------+-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 31     | zero_angle                      | get, set, save | float     | rad       | The encoder position the module considers to be 0 radians. Since this is an encoder position, zero_angle is constrained to [-pi, pi]. Unless obs_angular_displacement is set explicitly, this is the position regarded as 0 radians, and all displacements are measured in comparison to this point.                                               |
```

Comment:
This might be slightly confusing wording in the second part, regarding the "Unless obs_angular_displacement is set explicitly, this is the position regarded as 0 radians". This angle is always considered 0 radians in terms of angle commands, but it may not be 0 radiands of displacment if you set obs_angular_displacement. So maybe just add that "this is the position regarded as 0 radians of displacement", and maybe specify that angle commands ("ctrl_angle") they will always be referenced to this? Unless I'm wrong about that stuff, it can be hard to keep it all straight

Reply:
Just did a test to confirm on my desk. Here's what I did:
1. Observed angle was 2.6
2. Set zero angle with the sample button. Observed angle now 0
3. manually turn the motor by pi. observed angle pi. set ctrl_angle to 0. it goes to 0.
4. Coast the motor. manually turn to pi. set observed angle to 0. set ctrl_angle to 0. the motor locks right where it is, and does not go back to the originall set zero angle

Reply:
So, it's not always regarded as 0 displacement, and ctrl_angle isn't always in reference to this. the ctrl_angle is in reference to whatever is currently 0 displacement. To start, this is zero_angle, but that's not always true